docs/details/develop-checks/index.md

```diff
@@ -32,20 +32,245 @@ PMM uses Alertmanager API to get information about failed checks and show them o
+    - **lookback** (duration, optional): specifies how far in past to look back to metrics history. If this parameter is not specified, then query executed on the latest data. Example values: `30s`, `5m`, `8h`.
+    - **range** (duration, required): specifies time window of the query. This parameter is equal to [Prometheus API](https://prometheus.io/docs/prometheus/latest/querying/api/#range-queries).
+    - **step** (duration, required): query resolution. This parameter is equal to [Prometheus API](https://prometheus.io/docs/prometheus/latest/querying/api/#range-queries).
+- `POSTGRESQL_SELECT`
+    - **all_dbs** (boolean, optional): execute query on all available databases in PostgreSQL instance. If this parameter is not specified, then query executed on the default database (the one that was specified when service was added to PMM).
+
+## Develop checks
+
+!!! note alert alert-primary "Development / Debugging Only"
+Note that check development in PMM is currently for **debugging only** and **NOT for production use!**  Future releases plan to include the option to run custom local checks in addition to hosted Percona Platform checks.
+
+To develop custom checks for PMM:
+
+1. Install the latest PMM Server and PMM Client builds following the [installation instructions](https://www.percona.com/software/pmm/quickstart#).
+2. Run PMM Server with special environment variables:
+
+    - `PERCONA_TEST_CHECKS_FILE=/srv/custom-checks.yml` to use checks from the local files instead of downloading them from Percona Platform.
+    - `PERCONA_TEST_CHECKS_DISABLE_START_DELAY=true` to disable the default check execution start delay. This is currently set to one minute, so that checks run upon system start.
+    - `PERCONA_TEST_CHECKS_RESEND_INTERVAL=2s` to define the frequency for sending the SA-based alerts to Alertmanager.
+
+    ```sh
+    docker run -p 80:80 -p 443:443 --name pmm-server \
+    -e PERCONA_TEST_CHECKS_FILE=/srv/custom-checks.yml \
+    -e PERCONA_TEST_CHECKS_DISABLE_START_DELAY=true \
+    -e PERCONA_TEST_CHECKS_RESEND_INTERVAL=2s \
+    perconalab/pmm-server:dev-latest
+    ```
+
+3.  Log in to Grafana with credentials **admin/admin**.
+
+4. Go to **Configuration > Settings > Advanced Settings** and make sure the **Advisors** option is enabled.
+
+5.  Create `/srv/custom-checks.yml` inside the `pmm-server` container with the content of your check. Specify **dev** advisor in your check.
+
+6.  The checks will run according to the time interval defined on the UI. You can see the result of running the check on the home dashboard:
+
+    ![!](../../_images/HomeDashboard.png)
+
+7.  Click on the number of failed checks to open the Failed Checks dashboard:
+
+    ![!](../../_images/FailedChecks.png)
+
+8.  Go into the Docker container to output the logs of pmm-managed and read the check logs:
+
+```sh
+# get inside the container
+docker exec -it pmm-server bash
+# print and watch the logs
+supervisorctl tail -f pmm-managed
+```
```

Comment:
```suggestion
    ```sh
    # get inside the container
    docker exec -it pmm-server bash
    # print and watch the logs
    supervisorctl tail -f pmm-managed
    ```
```

Reply:
I noticed that p.8 is misaligned compared to the rest of code examples:
<img width="803" alt="Screenshot 2023-05-03 at 11 05 24" src="https://user-images.githubusercontent.com/81549/235862684-742deae6-1ba1-40f7-bb25-61f339fb9b2a.png">

Reply:
I wonder if the same can be achieved with just one command, i.e.:
`docker exec -it pmm-server supervisorctl tail -f pmm-managed`?

Reply:
Done